pkgs/by-name/st/STC/package.nix

```
@@ -0,0 +1,54 @@
+{ lib, stdenv, fetchFromGitHub, writeText }:
+
+stdenv.mkDerivation rec {
+  pname = "STC";
+  version = "v4.2";
```

*The version attribute must start with a digit e.g., "0.3.1rc2"*

```suggestion
  version = "4.2";
```